### PR TITLE
Ghc 8.8.1

### PR DESCRIPTION
**Pull request by SanchayanMaity**
Implements #16

### CODE REVIEW

**Review by chshersh — CHANGES_REQUESTED** [review]
Looks like you also need to bump up `base` version in the `.cabal` file. Otherwise looks good :+1:

**Comment by SanchayanMaity** [comment]
> Looks like you also need to bump up `base` version in the `.cabal` file. Otherwise looks good +1

Seems like bumping to 4.14 conflicts with haddock.

**Comment by chshersh** [comment]
@SanchayanMaity, yes, latest `pandoc` depends on `haddock-library-1.7` which doesn't support GHC-8.8. There's `haddock-library-1.8` which supports GHC-8.8, so we need to wait until `pandoc` updates its dependencies. Can take a while...

**Comment by SanchayanMaity** [comment]
@chshersh Understood. Will keep an eye out for the update.

**Comment by chshersh** [comment]
@SanchayanMaity I think you also should update `base` version in the `.cabal` file to enable `ghc-8.8.1`.

File: hakyll-shortcut-links.cabal [hunk 1]
```diff
@@ -31,6 +32,8 @@ common common-options
                        -fhide-source-paths
                        -Wmissing-export-lists
                        -Wpartial-fields
+  if impl(ghc >= 8.8.1)
+    ghc-options:       -Wmissing-deriving-strategies
```
Comment (chshersh) [inline hakyll-shortcut-links.cabal:36]: Let's error on missing deriving strategies, not just warn:

https://github.com/kowainik/tomland/blob/d9b7a1dc344e41466788fe00d0ea016f04629ade/tomland.cabal#L58-L60

**Comment by vrom911** [comment]
Hey @SanchayanMaity ! 
I went ahead and finished the issue in #22. Thank for your work on this again!

**Pull request closed without merging**